vulture/core.py

```diff
@@ -388,10 +393,18 @@ def visit_ClassDef(self, node):
             self.defined_classes, node.name, node, ignore=_ignore_class)
 
     def visit_FunctionDef(self, node):
```

Comment:
We also need to support async functions and classes.

Reply:
Since we are not discriminating between `async` functions and normal functions inside of Vulture and since both of them share `node.decorator_list`, therefore, I think that we already support them. In any case, I've also added a test case for an async function.

Also, I don't exactly get what do we mean by async classes? 😅

Reply:
I mean regular classes:

```
@foo
class Bar:
    pass
```